lib/hcrypto/rsa-ltm.c

```diff
@@ -132,7 +132,7 @@ ltm_rsa_private_calculate(mp_int * in, mp_int * p,  mp_int * q,
 {
     mp_err ret;
     mp_int vp, vq, u;
-    int where = 0; /* Ignore the set-but-unused warning from this */
+    int where HEIMDAL_UNUSED_ATTRIBUTE = 0; /* Ignore the set-but-unused warning from this */
```

Comment:
Please remove the `Ignore the set-but-unused warning from this` comment.

Reply:
Done